portfolio/src/main/webapp/script.js

```diff
@@ -15,13 +15,18 @@
 /**
  * Show/hide the description inside the element which has title as id.
  */
-function toggleContent(title) {
-  var descr = document.getElementById(title).getElementsByClassName("description");
-  if (descr && descr.length>0) {
-    if (descr[0].style.display === "none") {
-      descr[0].style.display = "block";
+function toggleContent(sectionId) {
+  if (document.getElementById(sectionId) == null) {
+      return;
+  }
+  const descriptionElement = document.getElementById(sectionId).getElementsByClassName("description");
+  if (descriptionElement && descriptionElement.length>0) {
+    const classes = descriptionElement[0].className.split(" ");
+    if (classes.indexOf("collapsed") === -1) {
+      classes.push("collapsed")
     } else {
-      descr[0].style.display = "none";
+      classes.splice(classes.indexOf("collapsed"), 1);
     }
+    descriptionElement[0].className = classes.join(" ");
```

Comment:
this a perfect example of functionality that you can extract into a separate function with a good name. Something like `toggleClass(element, class)`

Reply:
See also: https://sourcemaking.com/refactoring/smells/long-method

Reply:
Yeah, it makes sense.

Reply:
How can I check that the element parameter is a DOM element?